src/components/TopNav/UserMenu.vue

```diff
@@ -209,7 +209,11 @@ export default {
   computed: {
     getAvatarImage() {
       if (this.user.profile_photo) {
-        return process.env.apiUrl + this.user.profile_photo.file_url
+				if this.user.profile_photo.file_url.includes('http') {
+        	return this.user.profile_photo.file_url
+				} else {
```

Comment:
You don't need this else.

```js
if this.user.profile_photo.file_url.includes('http') {
    return this.user.profile_photo.file_url
}

return process.env.apiUrl + this.user.profile_photo.file_url
```

will suffice

I'd also check for the `://` bit, I'm sure sometimes normal urls contain the string http just by accident.